storybook/package.json

```diff
@@ -9,8 +9,9 @@
   "dependencies": {
     "@babel/core": "^7.3.4",
     "@nuxeo/nuxeo-elements": "3.0.0-SNAPSHOT",
+    "@nuxeo/nuxeo-testing-helpers": "3.0.0-SNAPSHOT",
     "@nuxeo/nuxeo-ui-elements": "3.0.0-SNAPSHOT",
-    "@polymer/iron-icon": "^3.0.1",
+    "@polymer/iron-icon": "^3.0.0",
```

Comment:
why ?

Reply:
Basically because it is the same (in terms of semver) and it cleans some lerna warnings.

Reply:
I agree it's the same (as long as we know 3.0.0 itself is working) but then you're not doing the same for other deps, hence the question. Anyway, not a blocker for sure ;)